complex-types.d.ts

```diff
@@ -1,10 +1,9 @@
 import type {Literal} from 'hast'
 
-export type Raw = {
+export interface Raw extends Literal {
```

Comment:
Please check your change locally, as I believe `npm test` will overwrite your changes. To keep them, include the `xo` ignore..

For why, see https://github.com/xojs/xo/releases/tag/v0.52.0.

Reply:
I see now, thanks. I did run `npm test`, but expected it to cause the script to fail and didn't think to check to see if it overwrote the files.